edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Edgelet/EdgeletCommandFactory.cs

```diff
@@ -20,14 +20,19 @@ public EdgeletCommandFactory(IModuleManager moduleManager, IConfigSource configS
             this.combinedConfigProvider = Preconditions.CheckNotNull(combinedConfigProvider, nameof(combinedConfigProvider));
         }
 
-        public Task<ICommand> CreateAsync(IModuleWithIdentity module, IRuntimeInfo runtimeInfo) =>
-            Task.FromResult(
-                CreateOrUpdateCommand.BuildCreate(
-                    this.moduleManager,
-                    module.Module,
-                    module.ModuleIdentity,
-                    this.configSource,
-                    this.combinedConfigProvider.GetCombinedConfig(module.Module, runtimeInfo)) as ICommand);
+        public Task<ICommand> CreateAsync(IModuleWithIdentity module, IRuntimeInfo runtimeInfo)
```

Comment:
I feel we should put this also behind a flag.. in case it regresses some scenario that we are not foreseeing.

Reply:
You're right, it doesn't hurt to be careful here. Adding a flag.